public/components/incontext_insight/generate_popover_body.tsx

```diff
@@ -302,7 +302,7 @@ export const GeneratePopoverBody: React.FC<{
                   aria-label="alert-assistant"
                   color="hollow"
                   size="m"
-                  type={getLogoIcon('gradient', shiny_sparkle)}
+                  type={getLogoIcon('white')}
```

Comment:
Nit: do we need to pass the fallback icon as before? Seems `shiny_sparkle` not been used in this file anymore. We can remove the import if we don't need it.